pkg/symbolizer/symbolizer.go

```diff
@@ -30,21 +30,21 @@ import (
 
 type Symbolizer struct {
 	logger    log.Logger
-	locations metastore.LocationStore
+	metaStore metastore.ProfileMetaStore
```

Comment:
Cosmetic: I'd prefer to have a separate interface here. It seems as if all this symbolizer needs would be:
```go
type SymbolizerMetaStore interface {
  GetSymbolizableLocations()
  Symbolize()
}
```
Happy to have that in some follow-up at some point. Just wanted to point out.

Reply:
Maybe we cab also avoid stutter?

Reply:
Usually, I would agree, but the `metastore.GetSymbolizableLocations` function expects a ProfileMetaStore as the second argument, and yeah there we could trim the interface down very slightly but it's almost identical at that point. I do think we should probably abstract the metastore further and rename the current metastore implementations to "drivers", then this suggestion will make sense again I think. I'd prefer to leave that for the future though.

Reply:
Sounds good to me!